src/components/PaymentForm.js

```diff
@@ -1,56 +1,59 @@
 import React, { Component } from "react";
-import { Form } from "semantic-ui-react";
+import { Form, Container } from "semantic-ui-react";
 
 export class PaymentFields extends Component {
   render() {
     return (
-      <Form id="payment-form">
-        <Form.Field>
-          <label>Select your subscription type:</label>
+      <Container>
+        <Form id="payment-form">
           <Form.Field>
-            <input type="radio" name="subscriptionType" />
-            Monthly - 5,000SEK/each
+            <label>Select your subscription type:</label>
+            <Form.Field>
+              <input type="radio" name="subscriptionType" />
+              Monthly - 5,000SEK/each
           </Form.Field>
-          <Form.Field>
-            <input type="radio" name="subscriptionType" />
-            Yearly - 10,000SEK/each
+            <Form.Field>
+              <input type="radio" name="subscriptionType" />
+              Yearly - 10,000SEK/each
+          </Form.Field>
+            <Form.Field>
+              <input type="radio" name="subscriptionType" />
+              Lifetime - 100,000SEK
```

Comment:
This may be out of scope for the story, but I remember that we need to refactor so that there is first just one subscription option.

Reply:
Yes this is being fixed on the next PR with the Stripe implementation